dist/api_data.py

```diff
@@ -910,6 +932,41 @@ def __ge__(self, other):
         type='category', subconfig=
         statistics_log_configuration_common)
 ]
+
+tiered_storage_configuration_common = [
+    Config('auth_timeout', 0, r'''
+        time in seconds that the authentication token is valid''',
+        min='0', max='100000'),
```

Comment:
In talking with Dennis on the cloud team, I don't think we need an auth timeout in the API.  The `auth_token` you have here will be an entry point into the KMS, and is long lived.  The KMS gives us actual keys to encrypt the message, which includes a timeout - giving the message a short shelf life.  If the `auth_token` ever needs to be changed, it will be instigated externally to us, with a `reconfigure` call.